main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-utils/v2/env"
+	. "github.com/bitrise-io/go-utils/v2/exitcode"
```

Comment:
I guess you used dot import here to avoid the `exitcode.Exitcode` eyesore. 😄

I remembered reading about this here: https://github.com/golang/go/wiki/CodeReviewComments#import-dot

It advices to only use this in a particular case where you have circular dependencies in your tests. I agree that this could make code harder to read but this step is so simple that it would not really hurt. But this step is also just a wrapper and all of the logic lives in a library. So probably we will not have to modify this step again and we could get away with `exitcode.Exitcode` too.

Reply:
This came up in [a different PR](https://github.com/bitrise-steplib/bitrise-step-restore-carthage-cache/pull/1#discussion_r1101049852), and I don't have a strong opinion about any of the two options, so I'll just follow the Go CodeReviewComments way. Heads up @vshah23